cpl.gemspec

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require_relative "lib/cpl/version"
+
+Gem::Specification.new do |spec|
+  spec.name    = "cpl"
+  spec.version = Cpl::VERSION
+  spec.authors = ["Justin Gordon", "Sergey Tarasov"]
+  spec.email   = ["justin@shakacode.com", "dzirtusss@gmail.com"]
```

Comment:
use sergey@shakacode.com

Reply:
Done in https://github.com/shakacode/heroku-to-control-plane/pull/4/commits/49ea29b5be5a9cccad978f49db0ac3f1206054d7.